embrace-android-features/src/main/kotlin/io/embrace/android/embracesdk/internal/ndk/EmbraceNativeCrashHandlerInstaller.kt

```diff
@@ -0,0 +1,108 @@
+package io.embrace.android.embracesdk.internal.ndk
+
+import android.os.Build
+import android.os.Handler
+import android.os.Looper
+import io.embrace.android.embracesdk.internal.SharedObjectLoader
+import io.embrace.android.embracesdk.internal.Systrace
+import io.embrace.android.embracesdk.internal.config.ConfigService
+import io.embrace.android.embracesdk.internal.crash.CrashFileMarkerImpl
+import io.embrace.android.embracesdk.internal.logging.EmbLogger
+import io.embrace.android.embracesdk.internal.logging.InternalErrorType
+import io.embrace.android.embracesdk.internal.storage.StorageService
+import io.embrace.android.embracesdk.internal.utils.Uuid
+import io.embrace.android.embracesdk.internal.worker.BackgroundWorker
+
+private const val HANDLER_CHECK_DELAY_MS = 5000
+
+internal class EmbraceNativeCrashHandlerInstaller(
```

Comment:
nit: perhaps this and future classes could be called NativeCrashHandlerInstaller**Impl**, or something else appropriate to how it's implemented?

Reply:
I don't like the Impl suffix, but I'd rather use the same for everything, and I don't really like the Embrace prefix either. Naming for interfaces and implementations is usually pretty bad 😅